.github/issue_labeler.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# PR template regexp's for issue labeler
+bug:
```

Comment:
You can also add `allocator` label to issues that contain `tbbmalloc` keyword.

Reply:
I saw the post about adding various labels regarding different components of TBB (https://github.com/oneapi-src/oneTBB/pull/1064#issuecomment-1505312363). I can add PR separately, adding an extra section to PR description, which differentiates 'type of PR' and 'component'.